plugins/org.obeonetwork.m2doc.sirius/src/org/obeonetwork/m2doc/sirius/commands/ExportRepresentationCommand.java

```diff
@@ -102,16 +100,17 @@ public ExportRepresentationCommand(TransactionalEditingDomain domain, List<Layer
     protected void doExecute() {
         if (this.layers.isEmpty() && this.representation instanceof DDiagram) {
             this.exportedDiagram = (DDiagram) this.representation;
+            if (!isRepresentationOpened) {
```

Comment:
Should this not be conditioned to the value of the 'refreshRepresentation' parameter?